support/util/util.go

```diff
@@ -354,42 +355,39 @@ func ApplyAWSLoadBalancerSubnetsAnnotation(svc *corev1.Service, hcp *hyperv1.Hos
 
 func DoesMgmtClusterAndNodePoolCPUArchMatch(mgmtClusterCPUArch, nodePoolArch string) error {
 	if mgmtClusterCPUArch != nodePoolArch {
-		return fmt.Errorf("multi-arch hosted cluster is not enabled and "+
+		return fmt.Errorf("hosted cluster is not capable of supporting multi-arch NodePools; "+
 			"management cluster and nodepool cpu architectures do not match; "+
-			"please use a multi-arch release image or a multi-arch release stream - management cluster cpu arch: %s, nodepool cpu arch: %s", mgmtClusterCPUArch, nodePoolArch)
+			"please use a multi-arch release image or a multi-arch release stream for the hosted cluster - management cluster cpu arch: %s, nodepool cpu arch: %s", mgmtClusterCPUArch, nodePoolArch)
 	}
 
 	return nil
 }
 
-func GetMgmtClusterCPUArch(ctx context.Context) (string, error) {
-	cfg, err := util.GetConfig()
-	if err != nil {
-		return "", err
+func GetKubeClientSet() (kubeclient.Interface, error) {
+	if os.Getenv("FAKE_CLIENT") == "true" {
```

Comment:
why is this needed?

Reply:
This was left behind while I was debugging the unit tests. This has been removed now.